src/Application.php

```diff
@@ -290,4 +277,61 @@ private function checkAndCompletionInputMapping(array $transformationConfig): ar
         }
         return $transformationConfig;
     }
+
+    private function sortByDependencies(array $transformationConfig): array
+    {
+        $rows = [];
+        foreach ($transformationConfig['rows'] as $row) {
+            $rows[$row['configuration']['phase']][$row['id']] = $row;
+        }
+        ksort($rows);
+
+        $result = [];
+        foreach ($rows as $phaseRows) {
+            $phaseResult = [];
+            $writeOutputRow = function (?array $writeBeforeValues = null, array $row) use (&$phaseResult): void {
+                if (!$writeBeforeValues) {
+                    $phaseResult[] = $row;
+                    return;
+                }
+                $beforeKey = null;
+                foreach ($writeBeforeValues as $writeBeforeValue) {
```

Comment:
máme nějaké závislé transformace, tak zjistím min. klíč ve výsledným poli před který musím transformaci uložit